sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosCatalogITest.scala

```diff
@@ -212,6 +220,236 @@ class CosmosCatalogITest extends IntegrationSpec with CosmosClient {
     containerProperties.getDefaultTimeToLiveInSeconds shouldEqual 5
   }
 
+  it can "select from a catalog table with default TBLPROPERTIES" in {
+    val databaseName = getAutoCleanableDatabaseName()
+    val containerName = RandomStringUtils.randomAlphabetic(6).toLowerCase + System.currentTimeMillis()
+    cleanupDatabaseLater(databaseName)
+
+    spark.sql(s"CREATE DATABASE testCatalog.$databaseName;")
+    spark.sql(s"CREATE TABLE testCatalog.$databaseName.$containerName (word STRING, number INT) using cosmos.items;")
+
+    val container = cosmosClient.getDatabase(databaseName).getContainer(containerName)
+    val containerProperties = container.read().block().getProperties
+
+    // verify default partition key path is used
+    containerProperties.getPartitionKeyDefinition.getPaths.asScala.toArray should equal(Array("/id"))
+
+    // validate throughput
+    val throughput = cosmosClient.getDatabase(databaseName).getContainer(containerName).readThroughput().block().getProperties
+    throughput.getManualThroughput shouldEqual 400
+
+    for (state <- Array(true, false)) {
+      val objectNode = Utils.getSimpleObjectMapper.createObjectNode()
+      objectNode.put("name", "Shrodigner's mouse")
```

Comment:
chain of quantum logical operator. If the Shrodigner's cat is alive it will eat the Shrodigner's mouse
so 
```
`Shrodigner's mouse`.isAlive  = not(`Shrodigner's cat`.isAllive)
```

just kidding :-)